src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -744,7 +744,7 @@ public void execute() throws GitException, InterruptedException {
                 } catch (GitException e) {
                     /* Nothing to do, just keeping defaultRemote = null */
                 }
-                
+
```

Comment:
No white space changes, please.  It complicates the review process for other proposed changes.

Reply:
sorry about that, just forcepushed to get rid of those